src/main/resources/data/htm/lang/de_de.json

```diff
@@ -0,0 +1,34 @@
+{
+  "text.htm.error.unlockable": "§cDieser Block kann nicht gesperrt werden",
+  "text.htm.error.no_lock": "§cDer Behälter ist nicht gesperrt",
+  "text.htm.error.not_owner": "§cDieser Behälter kann nicht gesperrt werden",
+  "text.htm.error.already_locked": "§cDieser Behälter ist bereits gesperrt",
+  "text.htm.error.lock_type": "§cUngültiger Sperrtyp",
+  "text.htm.error.flag_type": "§cUngültiger flag-Typ",
+  "text.htm.error.trust_self": "§cKann nicht sich selbst vertrauen",
+  "text.htm.error.already_trusted": "§r§a%s §cwurde bereits vertraut",
+  "text.htm.error.not_trusted": "§r§a%s §cwurde nicht vertraut",
+
+
+  "text.htm.select": "§bLinksklicke einen Block, um ihn auszuwählen",
```

Comment:
Linksklicken ist ein schlechter Imperativ für "left click", finde ich
Stattdessen "Linke Maustaste benutzen, um einen Block auszuwählen?"
So macht zumindestens das Wiki es
Im Tutorial (den Toasts) ist Linksklick ein Nomen (https://minecraft-de.gamepedia.com/Anleitungs-Hinweise), "Halte Linksklick fest" in dem Fall